$OpenBSD: patch-cmake_xapian-config_cmake_in,v 1.3 2019/10/23 12:13:35 sthen Exp $

Allow Xapian library to be found by CMake-based ports.

Index: cmake/xapian-config.cmake.in
--- cmake/xapian-config.cmake.in.orig
+++ cmake/xapian-config.cmake.in
@@ -1,5 +1,5 @@
 SET(prefix "@prefix@")
 SET(exec_prefix "@exec_prefix@")
-SET(XAPIAN_LIBRARIES "@libdir@/libxapian@LIBRARY_VERSION_SUFFIX@@SHLIBEXT@" CACHE FILEPATH "Libraries for Xapian")
+SET(XAPIAN_LIBRARIES "@libdir@/libxapian@LIBRARY_VERSION_SUFFIX@@SHLIBEXT@.${LIBxapian_VERSION}" CACHE FILEPATH "Libraries for Xapian")
 SET(XAPIAN_INCLUDE_DIR "@incdir@" CACHE PATH "Include path for Xapian")
 SET(XAPIAN_FOUND "TRUE")
